modules/calib3d/test/test_homography.cpp

```diff
@@ -144,7 +144,7 @@ void CV_HomographyTest::print_information_1(int j, int N, int _method, const Mat
     cout << "Type of srcPoints: "; if ((j>-1) && (j<2)) cout << "Mat of CV_32FC2"; else  cout << "vector <Point2f>";
```

Comment:
I have not tried this code and these tests myself, but I've got an impression that it prints a lot of info even if the tests succeed. Can we calm down the tests when it's all ok?